code/obj/item/clothing/uniforms.dm

```diff
@@ -805,6 +805,14 @@
 
 // Athletic Gear
 
+TYPEINFO(/obj/item/clothing/under/shorts)
+  random_subtypes = list(/obj/item/clothing/under/shorts,
```

Comment:
```suggestion
	random_subtypes = list(/obj/item/clothing/under/shorts,
```

Reply:
I'm really confused as to how spaces keep getting in here, it's not an issue I've had before.